ThePensionsRegulator.Frontend/HtmlGeneration/ComponentGenerator.TprSearchFooterLinks.cs

```diff
@@ -0,0 +1,76 @@
+﻿using GovUk.Frontend.AspNetCore;
+using GovUk.Frontend.AspNetCore.Extensions.HtmlGeneration;
+using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Mvc.Formatters;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ThePensionsRegulator.Frontend.HtmlGeneration
+{
+    public partial class ComponentGenerator
+    {
+        internal const string DefaultShowMoreQuestionsContent = "Show more questions";
+        public virtual TagBuilder GenerateTprSearchFooterLinks(TprSearchFooterLinks footerLinks)
+        {
+            var outer = new TagBuilder("footer");
+
+            var list = new TagBuilder("ul");
+            list.AddCssClass("govuk-list");
+            list.AddCssClass("tpr-search-results__links");
```

Comment:
Is this component 'TPR search' or 'TPR search results' or 'Q&As search results'? It seems unclear. CSS classes vary.

Reply:
You still refer to `tpr-search.js` in the ASP.NET example app page